src/M2ulPhyS.cpp

```diff
@@ -2603,8 +2606,8 @@ void M2ulPhyS::parseReactionInputs() {
 
       config.chemistryInput.reactionModels[r] = config.reactionModels[r];
       for (int p = 0; p < gpudata::MAXCHEMPARAMS; p++) {
-        config.chemistryInput.reactionModelParams[p + r * gpudata::MAXCHEMPARAMS] =
-            config.reactionModelParams[p + r * gpudata::MAXCHEMPARAMS];
+        // config.chemistryInput.reactionModelParams[p + r * gpudata::MAXCHEMPARAMS] =
```

Comment:
Maybe this should be deleted rather than commented out?

Reply:
Deleted.